src/customColors.js

```diff
@@ -0,0 +1,15 @@
+import { createTheme } from "@mui/material/styles";
+
+export const defaultTheme = createTheme();
+
+const customColors = {
+  blackMedium: "#1D2126",
+  blackLight: "#323232",
+  backgroundLight: "#FAFAFA",
+  greyDark: "#565656",
+  greyMedium: "#8F8F8F",
```

Comment:
I find using the `hsl` to be much easier to read and maintain different shades of colors. HSL stands for Hue, Saturation, and Lightness. These three grays can be represented as 
```jsx
grayDark: hsl(0, 0, 34),
grayMedium: hsl(0, 0, 56),
grayLight: hsl(0, 0, 91),
```

Notice how the lightness is the increasing from dark to light. Do you find this more readable as well?

Here's a blog post outlining the difference between hsl and hex colors if you're curious: https://blog.bitsrc.io/hex-vs-rgb-vs-hsl-what-is-the-best-method-to-set-css-color-property-f45d2debeee

Reply:
The team decided against using `hsl` for now, but I really like that system! Very good to know.

Reply:
Sounds good! Way to check in with the team.